README.md

```diff
@@ -101,6 +106,11 @@ spec:
 
 ##### Notes (SLO)
 
+- **metadata.labels:** *map[string]string|string[]* - optional field `key` <> `value`
```

Comment:
Shall we add one point with one / two examples in here, as I added for `annotations` [here](https://github.com/OpenSLO/OpenSLO/pull/64/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R115) so its easier in the future people to understand what goes where 🙏

Reply:
Great suggestions,  I have updated the spec with your suggestions and also added some example labels (`costCentre`, `userImpacting`, `team`)

Reply:
@geototti21 Please double check :)